circle.yml

```diff
@@ -32,9 +32,8 @@ test:
     override:
         # Disable incremental dexing on all gradle commands to speed up clean builds
         # https://sites.google.com/a/android.com/tools/tech-docs/new-build-system/2-5-alpha-gradle-plugin/trying-gradle-plugin-2-5
-        - ./gradlew checkstyle -Pandroid.useDexArchive=false
-        - ./gradlew lintDebug -Pandroid.useDexArchive=false
         - ./gradlew testDebugUnitTest -Pandroid.useDexArchive=false
+        - ./gradlew check -Pandroid.useDexArchive=false
```

Comment:
I think the reason that the static analysis went before the testing is so that the build would fail as early as possible if any of those checks failed. I think that should be maintained.